service/controller/resource/monitoring/prometheus/resource.go

```diff
@@ -257,6 +257,15 @@ func toPrometheus(ctx context.Context, v interface{}, config Config) (metav1.Obj
 		},
 	}
 
+	prometheus.Spec.Containers = []corev1.Container{
+		{
+			Name: "prometheus",
+			ReadinessProbe: &corev1.Probe{
+				InitialDelaySeconds: 300,
```

Comment:
Can you explain a bit why you set it to 5 minutes ? I'm not a big fan of magic numbers

Reply:
Well sorry, but that's a magic number.

Current readiness probe, fail after 10mn, so adding 5mn to it feels correct to me. Ideally this should be proportional to amount of stored data.

Reply:
Yes the number make sense :) My thinking here was that we could add a comment to explain where this is coming from :)

Reply:
Like why you set 300. I know we have to deal with those at times xD

Reply:
good point, comment added :)